src/components/atoms/MentorCard/style.js

```diff
@@ -0,0 +1,60 @@
+import styled, { keyframes } from 'styled-components';
+import { Body2, Heading2 } from '../../shared/Typography';
+
+const scaleAnimation = keyframes`
+  0% {
+    transform: scale(1);
+  }
+  100% {
+    transform: scale(0.95);
+  }
+`;
+const scaleDownAnimation = keyframes`
+  0% {
+    transform: scale(0.95);
+  }
+  100% {
+    transform: scale(1);
+  }
+`;
+export const MentorCard = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 500px;
+  height: 480px;
+  // &:hover {
+  //   animation: ${scaleAnimation} 0.3s ease-in-out forwards;
+  // }
+  // animation: ${scaleDownAnimation} 0.3s ease-in-out forwards;
+  border-radius: 10px;
+
+  img {
```

Comment:
![image](https://github.com/OpencodeNIT-R/project-kerberos/assets/100484401/48ef6efd-0a9d-4e65-b4e5-618624801427)
The aspect ratio is not being maintained, please use the proper object-fit.

Reply:
This slider needs to cover entire width.